### DISCUSSION TITLE
How i can get all qualitys ?

### DISCUSSION THREAD

**Discussion by 02losanger** [Tyrrrz/YoutubeExplode#541] [General]
How i can get all qualitys from video with fps

**Comment by Tyrrrz** [comment]
```csharp
var manifest = await youtube.Videos.Streams.GetManifestAsync("u_yIGGhubZs");
var qualities = manifest.GetVideoStreams().Select(s => s.VideoQuality).Distinct().ToArray();
```

**Reply by 02losanger** [reply]
![Ekran Alıntısı](https://user-images.githubusercontent.com/83074540/115904091-0eb3ff80-a46d-11eb-9042-5121867d844c.JPG)
 
i using framework 4.6.1 and explode 6.0.0 alpja2

**Reply by 02losanger** [reply]
Okey i fixed but how i can show a fps ?

because i cant download 1080p60 / 30 other

**Reply by Tyrrrz** [reply]
fps is in video quality:

https://github.com/Tyrrrz/YoutubeExplode/blob/afc4ee249590ddf15a5308b9fcbcda0337725da1/YoutubeExplode/Videos/Streams/VideoQuality.cs#L24-L27

**Reply by 02losanger** [reply]
i didnt understand but thank you :D

**Reply by Tyrrrz** [reply]
I'm just saying that to get FPS, you just need to do `VideoQuality.Framerate`. This will return fps.
